alarm/gpu-nvidia-tegra-k1/PKGBUILD

```diff
@@ -0,0 +1,194 @@
+        51-nvrm.rules
+        egl.pc
+        glesv2.pc)
+sha256sums=('f3539746e307751d0f6a0a9f827ae16a7514c5aeb95f43cc618a317aacb06f69'
+            'bdff89e517cd8cbba5f9e9d960bc79d7c2740bf563ff6aaab789a13a547f08fe'
+            '81ea8cf2c759edc19fbecdbaee3ddf22ab74d82e119da31ea9ddc8d1eb038ab4'
+            'ac63bec624005f948386f427ea7e2d679662146383b5fed27fddd6d8a327afa1')
+
+prepare() {
+  cd "${srcdir}"/Linux_for_Tegra/nv_tegra/
+  tar xjf nvidia_drivers.tbz2
+}
+
+package_gpu-nvidia-tegra-k1-nvrm() {
+  pkgdesc="NVIDIA binary nvrm daemon and libraries for Tegra K1"
+  depends=('linux-tegra-k1')
+  provides=('gpu-nvidia-tegra-k1="${pkgver}"')
+  conflicts=('gpu-nvidia-tegra-k1')
+
+  cd "${srcdir}/Linux_for_Tegra/nv_tegra"
+
+  mkdir -p "${pkgdir}${_install_prefix}/lib"
+
+  # copy other tegra specific libs
+  for lib in  libnvapputil.so \
+              libnvavp.so \
+              libnvdc.so \
+              libnvddk_2d_v2.so \
+              libnvddk_vic.so \
+              libnvmm.so \
+              libnvmm_camera_v3.so \
+              libnvmm_contentpipe.so \
+              libnvmm_parser.so \
+              libnvmm_utils.so \
+              libnvmmlite.so \
+              libnvmmlite_audio.so \
+              libnvmmlite_image.so \
+              libnvmmlite_utils.so \
+              libnvmmlite_video.so \
+              libnvodm_imager.so \
+              libnvodm_query.so \
+              libnvos.so \
+              libnvparser.so \
+              libnvrm.so \
+              libnvrm_graphics.so \
+              libnvsm.so \
+              libnvtestio.so \
+              libnvtestresults.so \
+              libnvtnr.so \
+              libnvtvmr.so \
+              libnvwinsys.so \
+              libtegrav4l2.so
+  do
+    cp usr/lib/arm-linux-gnueabihf/tegra/${lib} "${pkgdir}${_install_prefix}/lib"
+  done
+
+  for lib in  libnvidia-eglcore.so \
+              libnvidia-glcore.so \
+              libnvidia-glsi.so \
+              libnvidia-rmapi-tegra.so \
+              libnvidia-tls.so
+  do
+    cp usr/lib/arm-linux-gnueabihf/tegra/"${lib}.${_nv_version}" "${pkgdir}${_install_prefix}/lib/"
+    ln -s "${_install_prefix}/lib/${lib}.${_nv_version}" "${pkgdir}${_install_prefix}/lib/${lib}"
+  done
+
+  # copy license
+  mkdir -p "${pkgdir}${_install_prefix}/share/licenses/nvidia-tegra-k1"
+  cp LICENSE "$pkgdir${_install_prefix}/share/licenses/nvidia-tegra-k1/"
+
+  mkdir -p "${pkgdir}/etc/ld.so.conf.d"
+  echo "${_install_prefix}/lib" > "${pkgdir}/etc/ld.so.conf.d/$pkgbase.conf"
+
+  # copy udev rules
+  mkdir -p "${pkgdir}/usr/lib/udev/rules.d/"
+  install -D -m0644 "${srcdir}"/51-nvrm.rules "${pkgdir}/usr/lib/udev/rules.d/51-nvrm.rules"
+
+}
+
+package_gpu-nvidia-tegra-k1-x11() {
+  pkgdesc="NVIDIA binary OpenGL|ES libraries and X11 drivers for Tegra K1"
+  depends=('gpu-nvidia-tegra-k1="${pkgver}"' 'X-ABI-VIDEODRV_VERSION=19')
```

Comment:
Shall I remove the entire gpu-nvidia-tegra-k1 PKGBUILD, or just the x11 driver? Until the update to xorg-server 1.18 about a month ago, this package was working fine.